runtimes/runtimes/util/loggingUtil.ts

```diff
@@ -0,0 +1,68 @@
+import { Logging } from '../../server-interface'
+import { Connection } from 'vscode-languageserver/node'
+
+export type LogLevel = 'error' | 'warn' | 'info' | 'log' | 'debug'
+
+const logLevelMap: Record<LogLevel, number> = {
+    error: 0,
+    warn: 1,
+    info: 2,
+    log: 3,
+    debug: 4,
+}
+
+export const isLogLevelEnabled = (l1: LogLevel, l2: LogLevel): boolean => {
+    return logLevelMap[l1] >= logLevelMap[l2]
+}
+
+export const isValidLogLevel = (level: LogLevel): boolean => {
+    return level && typeof level === 'string' && ['fatal', 'error', 'warn', 'info', 'log', 'debug'].includes(level)
```

Comment:
First two checks are redundant.

Also cleaner solution IMO:

```typescript
export type LogLevel = 'error' | 'warn' | 'info' | 'log' | 'debug';

const logLevels: Set<LogLevel> = new Set(['error', 'warn', 'info', 'log', 'debug']);

function isLogLevel(level: string): level is LogLevel {
  return logLevels.has(level as LogLevel);
}
```

Reply:
Or perhaps even better:
```typescript
const logLevels = [ 'error', 'warn', 'info', 'log', 'debug' ] as const;

export type LogLevel = (typeof logLevels)[number];
```

Reply:
Thanks Jurre, let me check and test on local & make the change. The response from getConfiguration is any though - I just need to cross check if that works fine with (2) approach you mentioned.